manim/utils/unit.py

```diff
@@ -0,0 +1,22 @@
+"""Implement the Unit class."""
+
+import numpy as np
+
+from .. import constants, config
+
+
+def m(val):
+    return val
+
+
+def px(val):
+    return val * config.frame_width / config.pixel_width
+
+
+def pct(val, axis):
```

Comment:
Just a nitpick (like, really, you can totally ignore this) : I don't like the term "axis" here, as it can be misleading to something related to a GraphScene which is not. 

Why not `direction`?

Reply:
Oh the `axis` name comes from the use of the constants `X_AXIS`, `Y_AXIS`, etc. Nothing to do with GraphScene :sweat_smile: 

I agree it's not a great name, but I don't think `direction` is better. I'm open to alternatives.

Reply:
Maybe `dimension` ? (not sure)

But if there is no better alternative, i'm fine with axis.

Reply:
My problem is that neither "direction" or "dimension" are words that necessarily have a length. This function is trying to compute the (percent) length of something.

Let's use `axis` for now and revisit later :sweat_smile: